### ISSUE TITLE
TPPI Tweaks dont affect server

### ISSUE THREAD

**Issue by mizz141** [TPPIC/Test-Pack-Please-Ignore-2C#6]
The TPPI Tweaks dont affect the server at ALL... I find that a little bit Confusing...

**Comment by mizz141** [comment]
Update: The Tweaks affect the Molecular Assembler but not the Digi Miner and Compressed Things...

**Comment by XxXYaJrAbXxX** [comment]
As far as I am aware, TPPI Tweaks doesn't affect the digi miner recipe, that is done through mekanism config.

**Comment by wormxwood** [comment]
I have this exact same problem. TPPI tweaks added a rec for the digital miner, and removed the default one, as well as removing the Atomic Alloy Metalurgic Infuser rec. Now both recs, show up, despite having the "tweaked" text and everything else indicating TPPItweaks is getting run. 

EDIT: I just verified with a brand new world, this TPPIC update does not properly run the TPPItweaks, as far as the digital miner and Atomic Alloy Metalurgic Infuser recipe go.

**Comment by Prospector** [comment]
Hmm. Is TPPITweaks' jar in the server files?

**Comment by mizz141** [comment]
Yes, it is. but the Recipes are still broken...

Broken=All Recipes are there including original ones...

**Comment by wormxwood** [comment]
Yes. I get the "tweaked" messages in the tooltips, but like mizz141 says, both recipes are still there. I just removed the offensive ones using Crafttweaker last night. I did confirm this issue on a brand new server build with no added mods or anything. Sounds dumb- but can you verify that the digital miner only shows one recipe for you? When you click the arrow in the box on NEI, that it doesn't then go to the second? My users didn't notice for almost a month because it doesn't show up unless you click the arrow.
